src/effects.js

```diff
@@ -392,6 +392,13 @@ function Animation( elem, properties, options ) {
 		animation.opts.start.call( elem, animation );
 	}
 
+	// attach callbacks from options
```

Comment:
Stupid nitpick: probably better to start with uppercase 'attach' -> 'Attach', I know it was like that before and linter didn't catch that for some reason...